### PR TITLE
added test for alignment_info.py

### PR DESCRIPTION
**Pull request by cat-bug**
also removed unused code, change the order of expected/actual in number of asserts

### CODE REVIEW

**Review by andrewprzh — CHANGES_REQUESTED** [review]
Looks good! Just a couple minor changes

File: src/alignment_info.py [hunk 1]
```diff
@@ -10,6 +10,8 @@
 from .common import get_read_blocks
 from .polya_verification import shift_polya, shift_polyt
 
+REGION_TO_CHECK_LEN = 12
```
[the remark this answers is not in the document]
Comment (cat-bug) [reply src/alignment_info.py]: done

File: src/common.py [hunk 2]
```diff
@@ -36,6 +37,26 @@ def increment(self):
...
+    skipped = 3
+    soft_clipping = 4
+    hard_clipping = 5
+    padding = 6
+    seq_match = 7
+    seq_mismatch = 8
+
+    @classmethod
+    def get_match_events(cls):
+        return [cls.match, cls.seq_match, cls.seq_mismatch]
```
[the remark this answers is not in the document]
Comment (cat-bug) [reply src/common.py]: done

File: src/common.py [hunk 3]
```diff
@@ -36,6 +37,26 @@ def increment(self):
...
+
+    @classmethod
+    def get_ins_del_match_events(cls):
+        return [cls.match, cls.insertion, cls.deletion, cls.seq_match, cls.seq_mismatch]
```
[the remark this answers is not in the document]
Comment (cat-bug) [reply src/common.py]: done

**Review by andrewprzh — APPROVED** [review]

**Pull request merged**